ci-operator/config/openshift/installer/openshift-installer-master.yaml

```diff
@@ -500,6 +500,25 @@ tests:
     cluster_profile: openstack-vh-mecha-central
     workflow: openshift-e2e-openstack-external-lb
   timeout: 6h0m0s
+- always_run: false
+  as: e2e-openstack-masterscale
+  optional: true
+  steps:
+    cluster_profile: openstack-vexxhost
+    test:
+    - as: openstack-test-cpms
+      cli: latest
+      commands: |
+        export OPENSTACK_CONTROLPLANE_FLAVOR_ALTERNATE="$(<"${SHARED_DIR}/OPENSTACK_CONTROLPLANE_FLAVOR_ALTERNATE")"
+        make e2e-presubmit
```

Comment:
@damdo @JoelSpeed do you know where the `make` package should be installed? It's missing from the CCPMSO image and I'm not sure where it's added for the other projects ie `cluster-api-actuator-pkg`.

Reply:
https://github.com/openshift/release/blob/master/ci-operator/config/openshift/api/openshift-api-master.yaml#L50-L58

This is another example of how it's been done, perhaps this pattern would be helpful for you?

Reply:
> Might have to review the testplatform docs or ask in their channel to see if they can help?

Asked a couple of times, no concrete answer yet.

> https://github.com/openshift/release/blob/master/ci-operator/config/openshift/api/openshift-api-master.yaml#L50-L58
> This is another example of how it's been done, perhaps this pattern would be helpful for you?

I'll experiment if I can't find anything else.